api-specifications/oauth-token-introspection/1.0-draft/oauth-token-introspection-api-1.0.yml

```diff
@@ -0,0 +1,119 @@
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0
+
+paths:
+  /token_info:
+    post:
+      description: Returns the meta information surrounding the token, including the accessible resources.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/tokenIntrospectionRequest"
+          application/x-www-form-urlencoded:
+            schema:
+              $ref: "#/components/schemas/tokenIntrospectionRequest"
+      responses:
+        "200":
+          description: POST response
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/tokenIntrospectionResponse"
+
+components:
+  schemas:
+    tokenIntrospectionRequest:
+      type: object
+      properties:
+        token:
+          type: string
+      required:
+        - token
+    tokenIntrospectionResponse:
+      type: object
+      properties:
+        active:
+          type: boolean
+          example: true
+        client_id:
+          description: The API Client's key
+          type: string
+          example: PS7B3HzEmysp5lK8ejqI0
+        namespace_prefixes:
+          type: array
+          items:
+            type: string
+          example:
+            - "uri://ed-fi.org"
+        education_organizations:
+          type: array
+          items:
+            type: object
+            properties:
+              education_organization_id:
+                type: integer
+                example: 255901
+              name_of_institution:
+                type: string
+                example: "Grand Bend ISD"
+              type:
+                type: string
+                example: "edfi.LocalEducationAgency"
+        assigned_profiles:
+          type: array
+          items:
+            type: string
+          example:
+            - "Sample-Profile-Resource-IncludeOnly"
+        claim_set:
+          type: object
+          properties:
+            name:
+              type: string
+              example: "Ed-Fi Sandbox"
+        resources:
+          type: array
+          items:
+            type: object
+            properties:
+              resource:
+                type: string
+                example: "/ed-fi/students"
+              operations:
+                type: array
+                items:
+                  type: string
+                example:
+                  - "Create"
+                  - "Read"
+                  - "Update"
+                  - "Delete"
+        services:
+          type: array
+          items:
+            type: object
+            properties:
+              service:
+                type: string
+                example: "identity"
+              operations:
+                type: array
+                items:
+                  type: string
+                example:
+                  - "Create"
+                  - "Read"
```

Comment:
Would it make sense to include any of the other optional response fields? For example, exp?

Reply:
@vimayya agreed to add `exp` to the response. I added it to this specification and we will add it to the ODS API in `ODS-6505`.